CSharpFunctionalExtensions/MaybeExtensions.cs

```diff
@@ -69,5 +69,14 @@ public static void Execute<T>(this Maybe<T> maybe, Action<T> action)
 
             action(maybe.Value);
         }
+
+        public static TOut Execute<T, TOut>(this Maybe<T> maybe, Func<T, TOut> func)
```

Comment:
Looks like `Map` method, not like `Execute` or `Select`

Reply:
`Select` == `Map` in C#. Found some conflicts merging this PR. Could you re-submit?

Reply:
@a-z-hub Ping! 
Is this still active?